### PR TITLE
utils: Null dereference amend

### PR DESCRIPTION
**Pull request by paranlee**
issue #1553 related Null dereference amend works.

Signed-off-by: Paran Lee <p4ranlee@gmail.com>

### CODE REVIEW

File: utils/kernel.c [hunk 1]
```diff
@@ -127,6 +127,9 @@ bool check_kernel_pid_filter(void)
 	bool ret = true;
 	char *filename = get_tracing_file("options/function-fork");
 
+	if (filename == NULL)
+		return false;
```
Comment (namhyung) [inline utils/kernel.c:131]: I'm not sure if it could return NULL.

File: utils/script.c [hunk 2]
```diff
@@ -41,6 +41,9 @@ enum script_type_t get_script_type(const char *str)
 {
 	char *ext = strrchr(str, '.');
 
+	if (ext == NULL)
+		return;
```
Comment (namhyung) [inline utils/script.c:45]: Return `SCRIPT_UNKNOWN`

Comment (paranlee) [reply utils/script.c:45]: ok, thank you!

File: utils/kernel.c (hunk 1 above)
Comment (paranlee) [reply utils/kernel.c:131]: Would it be better to remove that null check?

Comment (namhyung) [reply utils/kernel.c:131]: No, I think it's possible.  Please keep it.

Comment (paranlee) [reply utils/kernel.c:131]: I got it. ;)

**Comment by paranlee** [comment]
@namhyung Thank you for code review! my patch updated.

**Pull request merged**